<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectangularPrism</class>
 <widget class="QDialog" name="RectangularPrism">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="drawReflected">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Reflected</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Translation">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Scale_Factor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="drawRotated">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Rotated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Normal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Scale Factor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Translation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Scale Factor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting point (X)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawScaled">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Scaled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Translated</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Translation">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Rotating Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Translation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Rotating_Point">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="X_StartingPoint">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Rotating_Angle">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Scale_Factor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting point (Y)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotating angle (degrees)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Rotating Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Reflection_Type">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertical</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="Length">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rectangular Prism</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Y_StartingPoint">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Rotating_Point">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Reflection">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Height">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Width">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
